tests/docker/install_local_irods_client_packages_el8.sh

```diff
@@ -0,0 +1,2 @@
+# To test unreleased versions of iRODS, build the iRODS packages, uncomment the following line, and update the version as necessary.
+#yum -y --nogpgcheck localinstall /irods_packages/irods-devel-4.3.3-0.el8.x86_64.rpm /irods_packages/irods-icommands-4.3.3-0.el8.x86_64.rpm /irods_packages/irods-runtime-4.3.3-0.el8.x86_64.rpm
```

Comment:
`dnf` should be used instead of `yum` on EL8+. This applies to other usages of `yum` as well.

Reply:
Done